package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "prepare": "husky install",
     "start": "vite",
-    "start:test": "vite build --mode test && vite preview --port 8080",
+    "start:test": "vite build --mode test && vite preview --mode test --port 8080",
```

Comment:
I don't think `mode` is a param for `preview`: https://vitejs.dev/config/preview-options.html